components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java

```diff
@@ -403,48 +447,86 @@ private ResultSet getIdpQueryResultSet(Connection dbConnection, String sortedOrd
         return prepStmt.executeQuery();
     }
 
+    /**
+     * Add requested required attributes in the SQL query.
+     *
+     * @param sqlQuery           SQL query which need to be modify.
+     * @param requiredAttributes Required attributes which needs to be return.
+     * @return modified SQL query.
+     */
+    private String appendRequiredAttributes(String sqlQuery, List<String> requiredAttributes) {
+
+        if (!requiredAttributes.isEmpty()) {
```

Comment:
let's use ListUtils.isNotEmpty here. Then we can remove the null checks on the places where appendRequiredAttributes is being called.

Reply:
CollectionUtils right? ListUtils doesnt contain isNotEmpty method.